src/gt4sd/frameworks/enzeptional/processing.py

```diff
@@ -23,22 +23,41 @@
 #
 """enzeptional - data processing utilities."""
 
+import random
+import warnings
 from abc import ABC
-from typing import Generic, List, Optional, Tuple, TypeVar, Union
-
+from typing import Any, Dict, Generic, List, Optional, Tuple, TypeVar, Union
 import numpy as np
+import os
 import torch
 from tape.datasets import pad_sequences
 from tape.registry import registry
 from tape.tokenizers import TAPETokenizer
-from transformers import AutoModelWithLMHead, AutoTokenizer
+from transformers import (
+    AlbertForMaskedLM,
+    AlbertTokenizer,
+    AutoModelForMaskedLM,
+    BertForMaskedLM,
+    BertTokenizer,
+    PreTrainedTokenizerFast,
+    RobertaForMaskedLM,
+    RobertaTokenizer,
+    T5EncoderModel,
+    T5Tokenizer,
+    XLNetLMHeadModel,
+)
+
+warnings.simplefilter(action="ignore", category=FutureWarning)
+
+
+def device_claim(_):
```

Comment:
this function should be imported from here: https://github.com/GT4SD/gt4sd-core/blob/main/src/gt4sd/frameworks/torch/__init__.py